plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -535,95 +534,6 @@ public void testMultipleSplitsPerBucket()
         assertEquals(drainSplits(hiveSplitSource).size(), 17);
     }
 
-    @Test
-    public void testSplitsGenerationWithAbortedTransactions()
-            throws Exception
-    {
-        java.nio.file.Path tablePath = Files.createTempDirectory("TestBackgroundHiveSplitLoader");
```

Comment:
this is flaky because the temp directory is common across the two tests.

Just change the path instead of disabling.

Reply:
They fail even if I run them in isolation, one by one. So it's not due to shared state or concurrency. I filed this a few days ago to track it: https://github.com/trinodb/trino/issues/19555